authorizer-app-backend/src/main/java/org/radarbase/authorizer/resources/SourceClientResource.kt

```diff
@@ -76,6 +77,35 @@ class SourceClientResource(
         return authorizationService.getAuthorizationEndpointWithParams(type, callbackUrl);
     }
 
+    @DELETE
+    @Path("{type}/authorization/{serviceUserId}")
+    @NeedsPermission(Permission.Entity.MEASUREMENT, Permission.Operation.READ)
+    fun deleteAuthorization(@PathParam("serviceUserId") serviceUserId: String,
```

Comment:
I’m not sure if `DELETE` takes a body. Could the access token be passed as an (optional) query param?